easybuild/easyconfigs/s/shapeit/shapeit-2.r837.GLIBCv2.12_linux_x86_64.eb

```diff
@@ -0,0 +1,24 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Tarball"
+
+name = 'shapeit'
+version = '2.r837.GLIBCv2.12'
```

Comment:
Since they provide two different build for two different glibc versions, for the same SHAPEIT version, it's probably better to keep it, but then maybe as a version suffix?

```python
version = '2.r837`
versionsuffix = '.GLIBCv2.12'
...
sources = ['shapeit.v%(version)s%(versionsuffix)s.Linux.static.tgz']